olp-cpp-sdk-core/src/http/curl/NetworkCurl.h

```diff
@@ -333,6 +333,9 @@ class NetworkCurl : public olp::http::Network,
   /// network transfer.
   std::unique_ptr<std::mutex[]> ssl_mutexes_{};
 #endif
+
+  // Used to store if curl_global_init(0 was ssuccessfull on construction
```

Comment:
Should it be public and rendered to Doxygen? If yes, please change to:
```suggestion
  /**
   * Stores if `curl_global_init(0)` was ssuccessfull on construction.
   */
```
Please also specify what it stores (e.g. Stores data if...).

Reply:
That is internal code and is not related to public headers. It is in doxygen probably but i dont think that it should be there. Anyways that is a comment more for developers of SDK.

Fixed typos.